packages/wonder-blocks-dropdown/components/single-select.js

```diff
@@ -94,6 +99,12 @@ type Props = {|
      * element to access pointer event state.
      */
     opener?: (openerProps: OpenerProps) => React.Element<any>,
+
+    /**
+     * When this is true, the dropdown body shows a search text input at the
+     * top. The items will be filtered by the input.
+     */
+    isFilterable?: boolean,
```

Comment:
I wonder if we should allow the filter algorithm to be overridden?

There are some really handy filtering rules besides "contains case insensitive match" such as the filtering rules that VSCode has for finding files, like matching starts of words. For example, ABC would match "Always Be Closing".

Reply:
@lillialexis wrote a pretty awesome smart matching algorithm for our github CLI tools would be nice to use.

Reply:
I did write a filter function that does exactly that @somewhatabstract! https://github.com/Khan/our-lovely-cli/blob/develop/cli/pull-request/smart-suggest.js